### PR TITLE
Add 'Alut' package

### PR DESCRIPTION
**Pull request by drodin**
* I've followed [this guide](https://hunter.readthedocs.io/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

---

needs a fork of https://github.com/vancegroup/freealut in cpp-pm org

### CODE REVIEW

**Comment by drodin** [comment]
@rbsheth Hi. I need a fork of https://github.com/vancegroup/freealut in cpp-pm org. Thanks!

File: cmake/projects/Alut/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Alut
+    VERSION
+    1.1.0-469287b-p0
+    URL
+    "https://github.com/drodin/freealut/archive/hunter-1.1.0-469287b.tar.gz"
```
Comment (NeroBurner) [inline cmake/projects/Alut/hunter.cmake:13]: created tag: https://github.com/cpp-pm/freealut/releases/tag/1.1.0-469287b-p0 please update URL to point to the tag

**Review by NeroBurner — CHANGES_REQUESTED** [review]
fork created, please update URL to point to tag

File: cmake/projects/Alut/hunter.cmake (hunk 1 above)
Comment (drodin) [reply cmake/projects/Alut/hunter.cmake]: done

**Review by NeroBurner — APPROVED** [review]

**Comment by NeroBurner** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.25.6

**Pull request merged**